lib/modelLexer.mll

```diff
@@ -63,6 +63,8 @@ module LU = LexUtils.Make(O)
     (* for bell files *)
     | "instructions" -> INSTRUCTIONS
     | "default" -> DEFAULT
+    | "&&" -> LAND
```

Comment:
```suggestion
```
There is a difficulty here: the `do_check_keyword` function is called on lexemes that match the `name` definition (see about line 120):
```OCaml
let name  = '_' ? alpha (alpha|digit|'_' | '.' | '-')* '\''?
```
As a result, the function is not called on string `"&&"` and this lexeme has to be recognized explicitly in the main lexer as:
```OCaml
| '&&'   { f Operator "&" ; LAND }
```

Reply:
Very good point! Thanks  @maranget 

Initially, I was using the keyword `and` (and `or`). I tested and convinced myself that it works. Then came to the realization that we would be better off using the same notation as ocaml for logical operators, changed them to `&&` and `||` but I completely missed this.